examples/decoding/plot_haxby_full_analysis.py

```diff
@@ -23,10 +23,12 @@
 
 # load labels
 import numpy as np
-labels = np.recfromcsv(haxby_dataset.session_target[0], delimiter=" ")
+labels = np.recfromcsv(haxby_dataset.session_target[0], delimiter=" ",
+                       dtype=[('labels', 'U12'), ('chunks', '<i8')])
```

Comment:
I think dtype doesn't need to be mentioned here similarly to what is done in the other haxby examples.